src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Server/ServerPrototype.cs

```diff
@@ -321,6 +368,35 @@ public void SendDataToServer()
             {
                 Microsoft.SqlServer.Management.Smo.Server server = this.dataContainer.Server;
                 bool changesMade = false;
+                bool bAlterServerConfig = false;
```

Comment:
Could you remove the leading Bs from these new properties? It's an old bit of Hungarian notation that we don't need anymore